cloud/availability.rst

```diff
@@ -163,6 +166,66 @@ Such crashes are a nightmare. But crash recovery is a way to make sure
 you sleep well at night by making sure none of your or your customers
 data is lost and your downtime is minimal.
 
+.. _cloud_forking:
+
+Forking
+=======
+
+Forking a Citus Cloud formation makes a copy of the cluster's data at an instant in time and produces a new formation in precisely that state. It allows you to change, query, or generally experiment with production data in a separate protected environment. Fork creation runs quickly, and you can do it as often as you want without causing any extra load on the original cluster. This is because forking doesn't query the cluster, rather it taps into the write-ahead logs for each database in the formation.
+
+How to Fork a Formation
+-----------------------
+
+Citus Cloud makes forking easy. The control panel for each formation has a "Fork" tab. Go there and enter the name, region, and node sizing information for the destination cluster.
+
+.. image:: ../images/cloud-fork.png
+
+Shortly after you click "Fork Formation," the new formation will appear in the Cloud console. It runs on separate hardware and your database can connect to it in the :ref:`usual way <connection>`.
+
+When is it Useful
+-----------------
+
+A fork is a great place to do experiments. Do you think that denormalizing a table might speed things up? What about creating a roll-up table for a dashboard? How can you persuade your manager that you need more RAM in the coordinator node rather than in the workers? You could prove yourself if only you could try your idea on the production data.
```

Comment:
Maybe say "persuade your colleagues" rather than "persuade your manager" (which implies a certain person reading this)